tests/spec/fs.unlink.spec.js

```diff
@@ -104,3 +104,14 @@ describe('fs.unlink', function() {
     });
   });
 });
+
+
+describe('fs.promises.unlink', function () {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function () {
+    var fs = util.fs();
+    expect(fs.promises.unlink).to.be.a('function');
+  });
+});
```

Comment:
Hello dleung25,

Just a heads up that Git is flagging the last line of your commit because it's expecting a newline at the end of the file.
Cheers (: .

Reply:
thanks for letting me know, I just added that in now